userguide/system.rst

```diff
@@ -934,7 +938,7 @@ Configuring email for TLS/SSL email providers is described in
 <https://forums.freenas.org/index.php?threads/are-you-having-trouble-getting-freenas-to-email-you-in-gmail.22517/>`__.
 
 
-.. note:: The %brand% user who receives periodic email can be set with
+.. note:: The %brand% user who receives periodic email is set with
    :menuselection:`System --> Advanced` in the
```

Comment:
...is set in the :guilabel:`Periodic Notification User` field in
:menuselection:`System --> Advanced`.